### ISSUE TITLE
Repeated "no primary server available" error

### ISSUE THREAD

**Issue by evandeininger** [microsoft/vscode-cosmosdb#1725]
Does this occur consistently? Yes
Repro steps:

Our CosmosDB has a firewall to only allow access from a VPN but when I disconnect it keeps giving me this error saying it's not available and only goes away when I uninstall this plugin. I'm cool with getting the error (obviously it makes sense to have it) but really only want it when I'm trying to connect or make a query, not get it over and over after dismissing the error notification.

1. disconnect from VPN
2. get this error
3. dismiss the error notification
4. repeat steps 2 and 3 until you turn VPN back on

Action: stat
Error type: MongoError
Error Message: no primary server available

Version: 0.15.0
OS: darwin
OS Release: 19.6.0
Product: Visual Studio Code
Product Version: 1.50.1
Language: en

<details>
<summary>Call Stack</summary>

```
ReplSetState.pickServer mongodb/replset_state.js:780:12
Timeout._selectServer mongodb/replset.js:1142:43
Timeout._onTimeout async-listener/glue.js:188:31
listOnTimeout timers.js:531:17
processTimers timers.js:475:7
```

</details>

**Comment by neelip** [comment]
This should be fixed with the upcoming 0.16.0 release.

**Comment by evandeininger** [comment]
Thanks for looking into this!
